src/scores/probability/crps_impl.py

```diff
@@ -10,9 +10,8 @@
 import xarray as xr
 
 import scores.utils
-
-from .checks import coords_increasing
-from .functions import (
+from scores.probability.checks import coords_increasing
+from scores.probability.functions import (
```

Comment:
It would be preferable to have separate MRs for each of the formatting, the reorganising of docstrings, and the type hinting. I think there is too much in a single MR here, so issues in some areas risk holding up a lot of value in the other areas.

Reply:
I think that would be good to move the type hinting into another MR since it will probably take some more effort to refine and implement for other functions across the package. I'm happy to move that to another MR but I think it is a bit redundant to open another MR for linting, ect. The above change is separated into its own commit (d57555a) and isn't a linting change so that can its own MR but its so small that I think its appropriate to keep in this MR since the commit message accurately describes the change.

Reply:
Perfect, thank you.